.changelog/unreleased/features/provider/1340-cryptographic-equivocation-feature.md

```diff
@@ -0,0 +1,3 @@
+- Add the cryptographic verification of equivocation feature to the provider
+  (cf. ADR-005 & ADR-013). ([\#1340](https://github.com/cosmos/interchain-
```

Comment:
Link doesn't seem to show correctly.
<img width="940" alt="Screenshot 2023-11-16 at 11 14 04" src="https://github.com/cosmos/interchain-security/assets/8606729/561e722e-1147-4ae2-9549-8c03b80ba0c7">
Probably you need:
```suggestion
  (cf. ADR-005 & ADR-013). ([\#1340](https://github.com/cosmos/interchain-security/pull/1340))
```